views/about/about.hbs

```diff
@@ -1,312 +1,366 @@
-                                <img class="single-pdf-thumbnail" src="https://schul-cloud-hpi.s3.hidrive.strato.com/images/pdf/pdf-thumbnail-Schul-Cloud-in-Unterricht-und-Schulalltag.jpg" />
-                            </div>
-                        </a>
-                    </div>
-                    <div class="single-pdf col-xs-12 col-md-4" data-pdf="https://s3.hidrive.strato.com/schul-cloud-hpi/Willkommenordner/Begleitmaterial/Broschuere_HPI-Schul-Cloud-und-Lernen-4.0.pdf">
-                        <a href="#">
-                            <div class="single-pdf-wrapper">
-                                <b class="single-pdf-title">HPI Schul-Cloud und Lernen 4.0 - Best-Practice-Unterrichtsbeispiele für die Sekundarstufe</b>
-                                <br/>
-                                <img class="single-pdf-thumbnail" src="https://schul-cloud-hpi.s3.hidrive.strato.com/images/pdf/pdf-thumbnail-HPI-Schul-Cloud-und-Lernen-4.0.jpg" />
-                            </div>
-                        </a>
-                    </div>
-                </div> <!-- .row.embed-pdf-row -->
-            </div> <!-- .container.embed-pdf -->
+		<h2 class="h3 section-title center">Bildungswissenschaftliche Begleitung</h2>
+		<p>Zur pädagogisch-didaktischen Unterstützung des Projekts im Sinne der Entwicklung von Beispielen für die
+			Unterrichtsgestaltung
+			mit verfügbarem digitalen Lernmaterial aus der HPI Schul-Cloud und dem Erkenntnisgewinn über die digitale
+			Unterrichtsgestaltung
+			mit der HPI Schul-Cloud ist eine externe bildungswissenschaftliche Begleitung beauftragt. Zum einen führt
+			die
+			<a href="https://osf.io/bzu4t/?view_only=14ad7519b5a24af7a83cf40b43377142" target="_blank">Universität
+				Tübingen</a> eine wissenschaftliche
+			Befragung der Pilotschulen zu Potenzialen der HPI Schul-Cloud durch. Zum anderen gestaltet das
+			<a href="http://www.gei.de/abteilungen/schulbuch-als-medium/schools-in-the-cloud-wissenschaftliche-begleitforschung-der-schul-cloud-einfuehrung.html"
+				target="_blank">Georg-Eckert-Institut</a>
+			aufbauend auf den Ergebnissen von Unterrichtsbeobachtungen und Workshops mit den Pilotschulen ein
+			Nutzungskonzept für die pädagogisch-didaktisch sinnvolle Anwendung der HPI Schul-Cloud. Außerdem hat das HPI
+			zusammen mit der <a href="https://www.presse.uni-augsburg.de/unipressedienst/2018/jan-maerz/2018_003/"
+				target="_blank">Universität Augsburg</a>
+			einen MOOC zu dem Thema <a href="https://mooc.house/courses/lernen40-2018" target="_blank">Lernen 4.0 -
+				Möglichkeiten und Grenzen einer Digitalisierung im Bildungsbereich</a> durchgeführt.</p>
+		<br />
+		<div class="embed-pdf">
+			<div class="row embed-pdf-row">
+				<div class="single-pdf col-xs-12 col-md-4">
+					<a href="https://schul-cloud-hpi.s3.hidrive.strato.com/Willkommenordner/Begleitmaterial/Broschuere_Die-Schul-Cloud-im-Unterricht-Fachuebergreifende-Unterrichtsszenarien-und-Methoden.pdf"
```

Comment:
url wirde von `data-pdf` in den inneren `a`-Tag verschoben (um die Semantik zu verbessern) + `target="blank"` hinzugefügt. Rest ist eigentlich nur linting